app/src/main/java/com/x8bit/bitwarden/ui/auth/feature/completeregistration/CompleteRegistrationScreen.kt

```diff
@@ -153,84 +150,198 @@ fun CompleteRegistrationScreen(
+            onValueChange = handler.onPasswordHintChange,
+            hint = stringResource(
+                R.string.bitwarden_cannot_recover_a_lost_or_forgotten_master_password,
+            ),
+            modifier = Modifier
+                .testTag("MasterPasswordHintLabel")
+                .fillMaxWidth(),
+        )
+        BitwardenClickableText(
+            label = stringResource(id = R.string.learn_about_other_ways_to_prevent_account_lockout),
+            onClick = handler.onLearnToPreventLockout,
+            style = nonMaterialTypography.labelMediumProminent,
+        )
+        Spacer(modifier = Modifier.height(24.dp))
+        BitwardenSwitch(
+            label = stringResource(id = R.string.check_known_data_breaches_for_this_password),
+            isChecked = isCheckDataBreachesToggled,
+            onCheckedChange = handler.onCheckDataBreachesToggle,
+            modifier = Modifier.testTag("CheckExposedMasterPasswordToggle"),
+        )
+        Spacer(modifier = Modifier.height(24.dp))
+        BitwardenFilledButton(
+            label = callToActionText,
+            isEnabled = nextButtonEnabled,
+            onClick = handler.onCallToAction,
+            modifier = Modifier.fillMaxWidth(),
+        )
+    }
+}
+
+@Composable
+private fun CompleteRegistrationContentHeader(
+    modifier: Modifier = Modifier,
+    configuration: Configuration = LocalConfiguration.current,
+) {
+
+    if (configuration.isPortrait) {
+        Column(
+            modifier = modifier,
+            horizontalAlignment = Alignment.CenterHorizontally,
+        ) {
+            OrderedHeaderContent()
+        }
+    } else {
+        Row(
+            modifier = modifier,
+            verticalAlignment = Alignment.CenterVertically,
+        ) {
+            OrderedHeaderContent()
+        }
+    }
+}
+
+/**
+ * Header content ordered with the image "first" and the text "second" which can be placed in a
+ * [Column] or [Row].
+ */
+@Composable
+private fun OrderedHeaderContent() {
+    Image(
+        painter = rememberVectorPainter(id = R.drawable.lock),
+        contentDescription = null,
+        modifier = Modifier.size(100.dp),
+    )
+    Spacer(modifier = Modifier.size(24.dp))
+    Column(
+        horizontalAlignment = Alignment.CenterHorizontally,
+    ) {
+        Text(
+            text = stringResource(R.string.choose_your_master_password),
+            style = MaterialTheme.typography.titleMedium,
+            color = MaterialTheme.colorScheme.onSurface,
+            textAlign = TextAlign.Center,
+        )
+        Spacer(modifier = Modifier.height(8.dp))
+        Text(
+            text = stringResource(
+                R.string.choose_a_unique_and_strong_password_to_keep_your_information_safe,
+            ),
+            style = MaterialTheme.typography.bodyMedium,
+            color = MaterialTheme.colorScheme.onSurface,
+            textAlign = TextAlign.Center,
+        )
+    }
+}
+
+@PreviewScreenSizes
```

Comment:
What is this guy?

Reply:
Gives previews for multiple sizes/form factors (desktop, landscape phone, foldable, etc)

Reply:
Nice, that's pretty cool!